File: Java/Q9.java

import java.util.Scanner;

class Q9 {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter a number: ");
        double num1 = sc.nextDouble();
        System.out.println("Enter a another number: ");
        double num2 = sc.nextDouble();
        System.out.println("Enter a operator: ");
        char operators = sc.next().charAt(0);

        switch (operators) {
            case '+' ->
                System.out.println("Sum: " + (num1 + num2));
            case '-' ->
                System.out.println("Sub: " + (num1 - num2));
            case '*' ->
                System.out.println("Mul: " + (num1 * num2));
            case '/' ->
                System.out.println("Div: " + (num1 / num2));
            default ->
                System.out.println("Invalid operator");
        }

    }
}

File: Python/Q9.py
num1 = int(input("Enter the number: "))
num2 = int(input("Enter the another number: "))
operators = input("Enter the operator: ")

match operators:
     case "+": print("Addition: ",num1 + num2)
     case "-": print("Subtraction: ",num1 - num2)
     case "*": print("Multiply: ",num1 * num2)
     case "/": print("Divide: ",num1 / num2)
     case _: print("Invalid")